stats/gRPC.md

```diff
@@ -96,8 +98,11 @@ All server metrics should be tagged with the following.
 
 `grpc_server_method` is set when an incoming request starts and is available in the context for
 the entire RPC call handling. 
+
 `grpc_server_status` is set when an incoming request finishes and is only available around metrics
 recorded at the end of the incoming request.
+Status codes should be stringified according to:
+https://github.com/grpc/grpc/blob/master/doc/statuscodes.md
```

Comment:
I would use this link:
https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto

Reply:
I considered that, but since this document is all about gRPC I think it's better to point to gRPC docs.

Reply:
I am fine either way.